java/ql/test/library-tests/dataflow/sign-analysis/SignAnalysis.ql

```diff
@@ -18,4 +18,5 @@ string getASignString(Expr e) {
 }
 
 from Expr e
+where not e instanceof Element or e.(Element).fromSource()
```

Comment:
```suggestion
where e.getEnclosingCallable().fromSource()
```
An `Expr` is actually always an `Element` (even though that's only apparent in the dbscheme), so I'd use the above instead.